packages/worker/src/prevent-discard.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  name: 'preventDiscard',
+  load: client => {
+    client.addOnError(event => {
+      event.errors[0].stacktrace = event.errors[0].stacktrace.map(function (frame) {
+        frame.file = frame.file.replace(/chrome-extension:/g, 'chrome_extension:')
+        frame.file = frame.file.replace(/moz-extension:/g, 'moz_extension:')
+        frame.file = frame.file.replace(/safari-extension:/g, 'safari_extension:')
+        frame.file = frame.file.replace(/safari-web-extension:/g, 'safari_web_extension:')
+        return frame
+      })
+    }, true)
```

Comment:
This can be simpler & more efficient by mutating in place and using 1 regex:

```suggestion
    const extensionRegex = /^(chrome|moz|safari|safari-web)-extension:/

    client.addOnError(event => {
      event.errors[0].stacktrace.forEach(function (frame) {
        frame.file = frame.file.replace(extensionRegex, '$1_extension:')
      })
    }, true)
```